### ISSUE TITLE
Querier not getting the UICArtPen

### ISSUE THREAD

**Issue by CanDKD** [uic-utah/uic-forms#18]
https://github.com/agrc/uic-forms/blob/0e40a9b1db1fc6b7fb552f1ff918b69c588ef34a/uic-forms/services/Querier.cs#L168-L221

This appeared to be working on Sept 12, 2018 when I ran the tool for period Oct 1, 2017 to March 31, 2018 and we were writing to the pdf forms.  Does not seem to work now when we are writing to csv file.  Steve can you check your local instance against the current development instance to make sure we have the same UICArtPen wells?

**Comment by steveoh** [comment]
What dates are you using and what is the object id of the artpen that is not being picked up?

**Comment by steveoh** [comment]
Is this still an issue?

**Comment by CanDKD** [comment]
I will create some ArtPen Wells in Dev with various parameters set to capture the queries then run uic-forms.exe against DEV.  I'll get back to you.

**Comment by CanDKD** [comment]
Steve, I created 2 UICArtPen wells in DEV that should be captured by uic-forms.exe.  I tried to run the most recent version dated 10/22 against 'staging' but it crashes.  I think we have recreated DEV since that time.  here is the latest log.txt file .

[7520-20191104.log.txt](https://github.com/uic-utah/uic-forms/files/3805952/7520-20191104.log.txt)

**Comment by steveoh** [comment]
looks like it wasn't able to connect... 2019-11-04 13:19:51.439 -07:00 [Verbose] Connecting to UDEQ...

**Comment by steveoh** [comment]
The staging database isn't versioned right now. That would cause this tool to fail as it needs the versioned views. That isn't the issue here probably but it's a good thing to note as a requirement.

**Comment by steveoh** [comment]
@CanDKD i'm going to need more information. The artpens are used in 8 query locations. The query I ran found items so can you provide me the test case data where you are expecting the artpens to be found. Thanks!

**Comment by CanDKD** [comment]
I think we may have a problem traversing the M:N relationship between AOR and ArtPen.  I checked the AOR Validation> no artificial penetrations query in the UIC Tool and that isn't working correctly either.  Here is a screenshot of the facility, well, authorization and artpens I created in DEV to check the uic-forms.exe and query.

![image](https://user-images.githubusercontent.com/4369080/69753094-ec02a780-110f-11ea-9693-02292253099a.png)

Can you view the bridging table for this relationship?  I can't see it.  If nothing is in that table I need to build it manually.

Maybe 10 hours isn't enough to sort this out.

**Comment by steveoh** [comment]
Did you RNP these features to default? I only see one artpen connected as uic admin?

**Comment by steveoh** [comment]
So after some digging, it appears that the view can't see the data because it has not been compressed?

@michaelagrc, at your next convenience, would you compress uic dev?

**Comment by CanDKD** [comment]
Yes, I ran rnp-delete.bat then ran uic-forms.exe.  I don't recall that compression was an issue in that past because I have successfully run the exe without first compressing.  I have also run the .exe  in Pro and reproduced the results I sent to EPA in Oct.  At that time I did not have any UICArtPen wells to report.  I thought we had this UICAoR / UICArtPen M:N issue resolved in the past.  I still think there is an issue with how we migrated Prod to Pro with the M:N relationship.
